### PR TITLE
fix: deprecate python36 from chalice

### PR DESCRIPTION
**Pull request by ShubhamChaturvedi7**
*Issue #, if available:*

*Description of changes:*
 `decrypt oracle` uses Code Pipeline to create the associated stack. The underlying tool is chalice which was using python36 codebuild base image. This PR pins the image to `aws/codebuild/standard:5.0` and python3.

*Testing*
Dev account and fork.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

# Check any applicable:
- [ ] Were any files moved? Moving files changes their URL, which breaks all hyperlinks to the files.

### CODE REVIEW

**Review by ajewellamz — APPROVED** [review]

File: decrypt_oracle/tox.ini [hunk 1]
```diff
@@ -35,7 +35,7 @@ envlist =
 
 
 [testenv:generate-pipeline]
-basepython = python3.9
+basepython = python3
```
Comment (texastony) [inline decrypt_oracle/tox.ini:38]: Revert: Python3.9 is the latest version of Python AWS Lambda supports.
It behooves us to use the latest Python available, 
so we do not have to re-deploy the Lambda again when Python3.7 or 3.8 goes EOL.
```suggestion
basepython = python3.9
```

File: decrypt_oracle/tox.ini [hunk 2]
```diff
@@ -44,7 +44,7 @@ commands = python .chalice/pipeline.py {posargs}
 
 
 [testenv:chalice-prep]
-basepython = python3.9
+basepython = python3
```
Comment (texastony) [inline decrypt_oracle/tox.ini:47]:
```suggestion
basepython = python3.9
```

File: decrypt_oracle/tox.ini [hunk 3]
```diff
@@ -59,7 +59,7 @@ commands =
     python {toxinidir}/.chalice/build-requirements.py
 
 [testenv:chalice]
-basepython = python3.9
+basepython = python3
```
Comment (texastony) [inline decrypt_oracle/tox.ini:62]:
```suggestion
basepython = python3.9
```

File: decrypt_oracle/tox.ini [hunk 4]
```diff
@@ -69,7 +69,7 @@ commands =
     chalice {posargs}
 
 [testenv:chalice-deploy]
-basepython = python3.9
+basepython = python3
```
Comment (texastony) [inline decrypt_oracle/tox.ini:72]:
```suggestion
basepython = python3.9
```

**Review by texastony — CHANGES_REQUESTED** [review]
Unless there is a very good reason,
I think the Oracle's Tox should always use python3.9.

**Review by texastony — APPROVED** [review]

**Pull request merged**